contents/handbook/growth/marketing/index.md

```diff
@@ -1,23 +1,41 @@
 ---
-title: Manifesto
+title: Overview
 sidebar: Handbook
 showTitle: true
 ---
 
-## Where does marketing fit at PostHog?
+Our approach is to provide internal Marketing-as-a-Service, aka MaaS. We are an engineering-led company, and believe in product-led, word of mouth growth. The best way marketing can contribute to PostHog's success is to act as a service to our product teams that amplifies the work they are doing.
 
-At the most fundamental level, we want to:
+## Marketing vision 
 
-1. Get developers and technical product managers to use PostHog - _features_
-2. Get those developers and technical product managers to be able to sell PostHog internally - _benefits_
+PostHog's [2026 vision](https://posthog.com/handbook/strategy/strategy#long-term-vision-for-2026) is to IPO in 2026 with $100m ARR. This is what we think marketing at PostHog will look like at that point. 
 
-Broadly speaking, the role of the _Marketing_ and _Website & Docs_ teams is to do 1. The role of the _Customer Success_ and _Growth_ teams is to do 2. 
+### Things we want to be brilliant at
 
-(This is a slightly simplified view due to natural overlaps, so consider this a rough approximation.)
+- **Behaviours**
+  - _Acting as a service to the engineering teams_ - actively engaged with engineering teams, finding out what they’re working on then partner with them to amplify what they are doing. Engineers don’t like talking to marketing people in most companies. We want our engineers to want to talk to marketing because we provide them with such a great service. 
```

Comment:
```suggestion
  - _Acting as a service to the engineering teams_ - we actively engage with engineering teams, find out what they’re working on and partner with them to amplify what they are doing. Engineers don’t like talking to marketing people in most companies. We want our engineers to want to talk to marketing because we provide them with such a great service. 
```